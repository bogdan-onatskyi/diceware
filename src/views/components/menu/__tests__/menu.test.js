import React from 'react';

import Menu from '../menu';

describe("Menu component", () => {
    it("should match saved snapshot", () => {
        const wrapper = shallow(
            <Menu/>
        );
        expect(wrapper).toMatchSnapshot();
    });

    it("should have 1 prop", () => {
        const propsStr = 'className';
        const props = [];
        for (let prop in Menu.propTypes) props.push(prop);
        expect(props.join(' ') === propsStr).toBe(true);
    });
});