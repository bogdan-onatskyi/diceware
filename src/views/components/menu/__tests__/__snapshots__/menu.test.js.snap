// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu component should match saved snapshot 1`] = `
<ul
  className="menu"
>
  <li
    className="menu__item"
    key="menu_1"
  >
    <Link
      className="icon-home"
      replace={false}
      to="/"
    >
      diceware пароли
    </Link>
  </li>
  <li
    className="menu__item"
    key="menu_2"
  >
    <Link
      className="icon-info"
      replace={false}
      to="/info"
    >
      Что это
    </Link>
  </li>
  <li
    className="menu__item"
    key="menu_3"
  >
    <Link
      className="icon-phone"
      replace={false}
      to="/contacts"
    >
      Контакты
    </Link>
  </li>
</ul>
`;
